### PR TITLE
codeintel: Add commit graph state to repo-specific uploads page

### PR DESCRIPTION
**Pull request by efritz**
This adds a banner on the code intelligence uploads page (repo only, not the site-admin area version) that says whether or not the repository's commit graph is fresh or stale, along with the time of the last refresh.

This should help us easily debug unexpected code intelligence results which are ephemeral because of a stale commit graph (where _processed_ uploads are not yet visible to the backend, though they appear in the UI as if they should be used).

Orthogonal efforts are being made to ensure that the commit graph update procedures are invoked as soon after an upload is processed as possible.

Screen Shot 2021-01-28 at 9 17 07 PM

Screen Shot 2021-01-28 at 9 16 53 PM

### CODE REVIEW

File: client/web/src/enterprise/codeintel/list/CodeIntelIndexesPage.story.tsx [hunk 1]
```diff
@@ -14,54 +14,52 @@ const { add } = storiesOf('web/codeintel/list/CodeIntelIndexesPage', module)
...
-                        failure: 'Whoops! The server encountered a boo-boo handling this input.',
-                        placeInQueue: null,
-                    }
-                )}
-            />
-        )}
-    </EnterpriseWebStory>
-))
+add('Page', () => {
+    const fetchLsifIndexes = fetch(
```
Comment (eseliger) [inline client/web/src/enterprise/codeintel/list/CodeIntelIndexesPage.story.tsx:18]: Does this really work? it still changes reference each time afaict 🤔

Comment (efritz) [reply client/web/src/enterprise/codeintel/list/CodeIntelIndexesPage.story.tsx:18]: Works fine :shrug:

Comment (efritz) [reply client/web/src/enterprise/codeintel/list/CodeIntelIndexesPage.story.tsx:18]: https://5f0f381c0e50750022dc6bf7-pusdfquaon.chromatic.com/?path=/story/web-codeintel-list-codeintelindexespage--page

**Review by Strum355 — APPROVED** [review]
Out of curiosity, how do we determine that its out of date? Is it possible for us to falsely claim to be up to date?

**Comment by efritz** [comment]
@Strum355 A repository commit graph is out of date if we trigger an update by either uploading a new index for that repository (in which case it's not visible to any commit and thus it's out of date by definition), or having a query for a commit we are unaware of (in which case we need to refresh the commit graph from gitserver).

**Review by slimsag — APPROVED** [review]
very cool

**Review by AlicjaSuska — COMMENTED** [review]
@efritz what do you think about using a [yellow warning alert](https://www.figma.com/file/HWLuLefEdev5KYtoEGHjFj/%F0%9F%93%9AWeb-Global-styles?node-id=572%3A74) for the 'stale' info? We usually use blue alerts for information (ex. how to do something, how something was generated, etc.) . This case, information in the alert to be more important for the user as they may be looking at the outdated state. @macraig wdyt?

**Pull request merged**